### PR TITLE
Create nl.js - Dutch (v3.8.1)

### PR DESCRIPTION
**Pull request by Semliot**

### CODE REVIEW

**Comment by mty22** [comment]
Perfect, thank you!

Please let me know work breakdown percentages.

I'll also request for another native speaker to review!

**Comment by Semliot** [comment]
> Perfect, thank you!
> 
> Please let me know work breakdown percentages.
> 
> I'll also request for another native speaker to review!

Semliot (50%) - Semliot#0070 - https://krunker.io/social.html?p=profile&q=Semliot
nickauthentic (25%) - nickauthentic#9999 - https://krunker.io/social.html?p=profile&q=NickAuthentic
gluzly (25%) - gluzly#9999 - https://krunker.io/social.html?p=profile&q=Gluzly

AdoreMe - AdoreMe#0082 - https://krunker.io/social.html?p=profile&q=AdoreMe

AdoreMe (native speaker) reviewed the file.
She just needs the role in KB.

**Comment by mty22** [comment]
> > Perfect, thank you!
> > Please let me know work breakdown percentages.
> > I'll also request for another native speaker to review!
> 
> Semliot (50%) - Semliot#70 - https://krunker.io/social.html?p=profile&q=Semliot
> nickauthentic (25%) - nickauthentic#9999 - https://krunker.io/social.html?p=profile&q=NickAuthentic
> gluzly (25%) - gluzly#9999 - https://krunker.io/social.html?p=profile&q=Gluzly
> 
> AdoreMe - AdoreMe#82 - https://krunker.io/social.html?p=profile&q=AdoreMe
> 
> AdoreMe (native speaker) reviewed the file.
> She just needs the role in KB.

Nice, this is the kind of quality I am looking for. Everything is done prior to pushing the pull request.

I've pinged a native speaker separate to your group just to do a once-over.

**Comment by Semliot** [comment]
> > > Perfect, thank you!
> > > Please let me know work breakdown percentages.
> > > I'll also request for another native speaker to review!
> > 
> > 
> > Semliot (50%) - Semliot#70 - https://krunker.io/social.html?p=profile&q=Semliot
> > nickauthentic (25%) - nickauthentic#9999 - https://krunker.io/social.html?p=profile&q=NickAuthentic
> > gluzly (25%) - gluzly#9999 - https://krunker.io/social.html?p=profile&q=Gluzly
> > AdoreMe - AdoreMe#82 - https://krunker.io/social.html?p=profile&q=AdoreMe
> > AdoreMe (native speaker) reviewed the file.
> > She just needs the role in KB.
> 
> Nice, this is the kind of quality I am looking for. Everything is done prior to pushing the pull request.
> 
> I've pinged a native speaker separate to your group just to do a once-over.

Sounds good!

**Comment by PlopperKR** [comment]
Hey, Can I still help with the dutch translation or is it done?

**Comment by PlopperKR** [comment]
> Hey, Can I still help with the dutch translation or is it done?

Or do I need to review it or anything?

**Comment by Semliot** [comment]
> > Hey, Can I still help with the dutch translation or is it done?
> 
> Or do I need to review it or anything?

It's already done and someone else is reviewing it.

**Comment by mty22** [comment]
@PlopperKR feel free to review if you wish to help =) click on the files changed tab and have a look there. If you find an issue, add a comment on the line that is a problem.

File: nl.js [hunk 1]
```diff
@@ -0,0 +1,747 @@
...
+settings.quality.rendDst Render Afstand
+settings.quality.res Resolutie
+settings.quality.part Particles
+settings.quality.mapDet Map Details
+settings.quality.noTex Geen Texturen
+settings.quality.partdist Particle Afstand
+settings.quality.renderdist Render Afstand
+settings.quality.reflection Reflectie Kwaliteit
+settings.quality.antiAlias Antialiasing
+settings.quality.shadows Schaduw
```
Comment (PlopperKR) [inline nl.js:127]: 'Schaduw' should be changed to schaduwen

File: nl.js [hunk 2]
```diff
@@ -0,0 +1,747 @@
...
+settings.quality.softShad Zachte Schaduw
```
Comment (PlopperKR) [inline nl.js:128]: 'Zachte Schaduw' should be changed to Zachte Schaduwen

File: nl.js [hunk 3]
```diff
@@ -0,0 +1,747 @@
...
+settings.gameprivacy.header Gameplay Privacy
+settings.gameprivacy.streamer Streamer Modus
+settings.gameprivacy.anon Anonieme Modus
+settings.gameprivacy.hideGameTitle Verstop Game Titel
+settings.gameplay.header Gameplay
+settings.gameplay.sensitivityX X Gevoeligheid
+settings.gameplay.sensitivityY Y Govoeligheid
+settings.gameplay.aimsensX Aim X Gevoeligheid
+settings.gameplay.aimsensY Aim Y Gevoeligheid
+settings.gameplay.challenge Challenge Modus
```
Comment (PlopperKR) [inline nl.js:255]: 'Challenge Modus' Challenge is translated into 'Uitdaging/Uitdagings' So 'Challenge Modus' Should be changed to Uitdagings Modus

File: nl.js [hunk 4]
```diff
@@ -0,0 +1,747 @@
...
+settings.audio.ambient Omringende Volume
+settings.audio.voice Stem Volume
+settings.audio.weps Wapen Volume
+settings.audio.player Speler Volume
+settings.audio.skin Cosmetica Volume
+settings.audio.ui UI Volume
+settings.audio.asset Asset Volume
+settings.audio.dialogue Actie Volume
+settings.viewmodel.header Bekijk model
+settings.viewmodel.fov Gezichtsveld
```
Comment (PlopperKR) [inline nl.js:290]: 'Gezichtsveld' is the outcome in Google Translate I figured. It should be Zichtveld

File: nl.js [hunk 5]
```diff
@@ -0,0 +1,747 @@
...
+settings.viewmodel.fps Wapen Gezichtsveld
```
Comment (PlopperKR) [inline nl.js:291]: 'Wapen Gezichtsveld' should be Zichtveld

File: nl.js [hunk 6]
```diff
@@ -0,0 +1,747 @@
...
+settings.viewmodel.bobbing Wapen Bobbing
+settings.viewmodel.leaning Wapen Leaning
```
Comment (PlopperKR) [inline nl.js:293]: 'Wapen Leaning' is not translated well. Leaning is 'Leunen' in Dutch. that should be changed to 'Wapen Leunen'

File: nl.js [hunk 7]
```diff
@@ -0,0 +1,747 @@
...
+windows.store.premium2 Krunker Premium
+windows.store.buyP Kies een weergavenaam en meer met
+windows.store.buy Koop en verkoop items op het
+windows.store.wheels Prijswielen
+windows.store.unlock Ontgrendel nieuwe wapenskins en items
+windows.store.amount {0} KR in account
+windows.store.login om items uit de winkel te kopen en te ontgrendelen!
+windows.store.purchasing Inkoop ...
+windows.store.unboxed {0} {1} geopend
+windows.trade.header Traden
```
Comment (PlopperKR) [inline nl.js:520]: 'Traden' is 'Ruilen' in Dutch

File: nl.js [hunk 8]
```diff
@@ -0,0 +1,747 @@
...
+social.market.unlisting Item Weghalen...
+social.market.posting Item Posten...
+social.market.req lvl {0} nodig
+social.market.cant Kan niet Veroorloven
+social.market.purchase Kopen
+social.market.fee Geen List Fee
+social.market.failed Mislukt. Probeer Opnieuw
+social.market.head Hoofd Item
+social.market.back Rug Item
+social.market.waist Waist Item
```
Comment (PlopperKR) [inline nl.js:678]: 'Waist' is 'Taille' in Dutch. It should be changed to 'Taille Item'

File: nl.js [hunk 9]
```diff
@@ -0,0 +1,747 @@
...
+social.market.face Gezicht Item
+social.market.shoe Schoen Item
+social.market.pet Huisdier Item
+social.market.melee Melee Item
+social.market.spray Spray Item
+social.market.dye Kleurstof Item
+social.market.none Geen Items Gevonden
+social.market.login Je moet ingelogd zijn.
+social.market.grace gereed in {0}
+social.market.sort Filteer/Sorteer
```
Comment (PlopperKR) [inline nl.js:688]: I found a spelling error here. 'Filteer' is misspelled, it should be Filter/Sorteer

**Comment by mty22** [comment]
@Semliot can you please review @PlopperKR 's comments?

File: nl.js (hunk 9 above)
Comment (Semliot) [reply nl.js:688]: 👍

File: nl.js (hunk 7 above)
Comment (Semliot) [reply nl.js:520]: Changed it.

File: nl.js (hunk 3 above)
Comment (Semliot) [reply nl.js:255]: Alright I will change that.

File: nl.js (hunk 5 above)
Comment (Semliot) [reply nl.js:291]: Denied. It doesn't sound right.

File: nl.js (hunk 6 above)
Comment (Semliot) [reply nl.js:293]: Denied.

File: nl.js (hunk 8 above)
Comment (Semliot) [reply nl.js:678]: I changed it.

File: nl.js (hunk 2 above)
Comment (Semliot) [reply nl.js:128]: Done.

File: nl.js (hunk 1 above)
Comment (Semliot) [reply nl.js:127]: Done.

**Comment by Semliot** [comment]
> @Semliot can you please review @PlopperKR 's comments?

I have reviewed everything.

**Comment by mty22** [comment]
image

**Review by mty22 — APPROVED** [review]

**Comment by mty22** [comment]
image
#verified chat in KB. 25% each.

**Pull request merged**